library/alloc/src/collections/btree/map.rs

```diff
@@ -2716,14 +2717,14 @@ impl<K, V, A: Allocator + Clone> BTreeMap<K, V, A> {
 ///
 /// A `Cursor` is like an iterator, except that it can freely seek back-and-forth.
 ///
-/// Cursors always point to an element in the tree, and index in a logically circular way.
-/// To accommodate this, there is a "ghost" non-element that yields `None` between the last and
-/// first elements of the tree.
+/// Cursors always point to a gao between two elements in the map, and can
```

Comment:
```suggestion
/// Cursors always point to a gap between two elements in the map, and can
```

Reply:
ahh sorry I didn't add the suggestion to the other two comments where the typo occured.